tutorials/vue/vue-table.md

```diff
@@ -0,0 +1,85 @@
+Vue makes it very easy to use tables in your projects because of its list rendering functionality.
+With a few lines of html and a `v-for` loop, you can have a table that is easy to change should you need
```

Comment:
Capitalize "HTML", and also let's just say "you can create a table with one row for every element in an array." Better to be as concrete as possible because "make additions" is a little too abstract.